core/commands/add.go

```diff
@@ -30,11 +30,9 @@ const (
 
 var AddCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline: "Add a file to ipfs.",
+		Tagline: "Add a file or directory structure to ipfs.",
```

Comment:
Would "hierarchy" be more fitting? I want it to be clear that the whole directory and its subdirectories will be added too (since `-r` must be present).